src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -25,24 +25,41 @@ EOF
 }
 
 function clone () {
-    curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+    if [ $TEST_TYPE == "local" ]; then
+        curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+    elif [ $TEST_TYPE == "kube" ]; then
+        kubectl exec -it $CODEWIND_POD_ID -- curl -H "Content-type: application/json" -d '{ "projectName": "'$1'" , "parentPath": "'$2'", "url": "'$3'" }' "https://localhost:9191/api/v1/projects" --insecure
+    fi
+
+    if [[ $? -ne 0 ]]; then
+        echo -e "${RED}Codewind was unable to clone the projects. ${RESET}\n"
+        exit 1
+    fi
 }
 
 function setup {
     DATE_NOW=$(date +"%d-%m-%Y")
     TIME_NOW=$(date +"%H.%M.%S")
-    BUCKET_NAME=turbine-$TEST_SUITE
-    TEST_DIR=$CW_DIR/src/pfe/file-watcher/server
+    BUCKET_NAME=turbine-$TEST_TYPE-$TEST_SUITE
+    TEST_DIR=$CW_DIR/src/pfe/file-watcher/server/test
```

Comment:
I am think of we may need to copy all contents under `server` because user may add more devDependencies for the tests, and devDependencies is in package.json under `server` not `server/test`, then for the following code, we need to do:
1) docker/kubectl cp firstly to copy contents under `server`
2) run `npm install --only=dev"` inside PFE container

Reply:
I just changed 

1. `TEST_DIR=$CW_DIR/src/pfe/file-watcher/server` to `TEST_DIR=$CW_DIR/src/pfe/file-watcher/server/test`
2. `docker cp $TEST_DIR/test $CODEWIND_CONTAINER_ID:$TEST_DIR_CONTAINER` to `docker cp $TEST_DIR $CODEWIND_CONTAINER_ID:$TEST_DIR_CONTAINER`

we were already copying test dir

Reply:
Yes, your change is for copying `server/test` dir which is same as before.

But I mean we **not only**  copy the `server/test` dir, we copy all contents under `server` dir instead because `package.json` is under `server` dir and user may need `devDependencies` in `package.json` for installing new packages that user add for testing. 

So basically we can do two things:
1 Copy the whole contents under `server` dir
2 Run `npm install --only=dev` inside PFE container **after** `docker/kubectl cp` (currently we run `npm install --only=dev` inside PFE container **before** `docker/kubectl cp`)